prime-router/build.gradle.kts

```diff
@@ -583,16 +583,24 @@ task<Exec>("uploadSwaggerUI") {
     commandLine("./upload_swaggerui.sh")
 }
 
+tasks.register("killFunc") {
+    exec {
+        workingDir = project.rootDir
```

Comment:
Not related, but this is a crude fix for the the issue that azure func no longer responds to `SIGINT`